File: python3/1.py
class Solution:
    def twoSum(self, nums: List[int], target: int) -> List[int]:
        numMap = {}
        for i in range(len(nums)):
            if target - nums[i] in numMap:
                return [numMap[target - nums[i]], i]
            numMap[nums[i]] = i

File: golang/1.go
func twoSum(nums []int, target int) []int {
    m := make(map[int]int)
    for idx, val := range nums {
        if i, ok := m[target - val]; ok {
            return []int{idx, i}
        }
        m[val] = idx
    }
    return []int{}
}